tracer/src/Datadog.Trace/ClrProfiler/DefinitionsIdAttribute.cs

```diff
@@ -0,0 +1,22 @@
+// <copyright file="DefinitionsIdAttribute.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.ClrProfiler
+{
+    internal class DefinitionsIdAttribute : Attribute
+    {
+        public DefinitionsIdAttribute(string definitionsId, string derivedDefinitionsId)
+        {
+            DefinitionsId = definitionsId;
+            DerivedDefinitionsId = derivedDefinitionsId;
+        }
+
+        internal string DefinitionsId { get; set; }
```

Comment:
nit: I think you can remove the `set` and it would still work

Reply:
yep thank you fixed in cb6ea0a31c2c1419d43b51ff3607cb64696bd9ad